packages/ethereum-contracts/truffle-config.js

```diff
@@ -385,7 +385,7 @@ const E = (module.exports = {
     // Configure your compilers
     compilers: {
         solc: {
-            version: "0.8.19", // Fetch exact version from solc-bin (default: truffle's version)
+            version: "0.8.21", // Fetch exact version from solc-bin (default: truffle's version)
```

Comment:
hardhat and truffle still like to download solc themselves unlike foundry. It's possible to configure and workaround, but hard to keep doing that across all project. Will need more creativity of using tooling.